AusTraits_tutorial.qmd

```diff
@@ -184,48 +192,103 @@ Like the core `traits` table, each is in 'long' format.
 
 The tables `locations`, `contexts`, `methods` and `taxa` include metadata that links seamlessly to individual rows within `traits`.
 
-| table     | metadata in table                                                                                                                               | columns that link to austraits\$traits                                                                                                                         |
-|--------------|----------------------------|-------------------------------|
-| locations | location name, location properties, latitude, longitude                                                                                         | dataset_id, location_id                                                                                                                                        |
-| contexts  | context name, context category (method context, temporal, entity context, plot, treatment), context property                                    | dataset_id, link_id (identifier to link to: method_context_id, temporal_id, entity_context_id, plot_id, treatment_id), link_vals (identifier value to link to) |
-| methods   | dataset description, dataset sampling strategy, trait collection method, data collectors, data curators, dataset citation, source_id & citation | dataset_id, trait_name, method_id                                                                                                                              |
-| taxa      | genus, family, scientific name, APC/APNI taxon concept/taxon name identifiers                             
+As such, you can use `dplyr::join` functions to join together the relational tables into a single (much) wider table. Some `{austraits}` package functions also join key columns to the traits table.
```

Comment:
Is it `plyr::join` not `dplyr::join`? That's what comes up with the help file

Reply:
I guess I was referring to the family of join functions in dplyr.